app/controllers/geocoder_controller.rb

```diff
@@ -157,7 +157,11 @@ def search_osm_nominatim_reverse
                     :type => object_type, :id => object_id)
     end
 
-    render :action => "results"
+    respond_to do |format|
+      format.html { render :action => "results" }
+      format.json { render :json => @results }
+      format.any { render :action => "results" }
```

Comment:
Presumably we don't really need the first line here if the render is just the same as the any case?